libr/bin/pdb/pdb_downloader.c

```diff
@@ -199,5 +212,5 @@ int r_bin_pdb_download(RCore *core, int isradjson, int *actions_done, SPDBOption
 	}
 	deinit_pdb_downloader (&pdb_downloader);
 
-	return 0;
+	return !ret;
```

Comment:
this is the usual int vs bool mess :/ I'm never sure what functions are supposed to return. Is 0 the right code? Is this function returning just 1 and 0? If so, can we make it return bool instead?

Reply:
@ret2libc yeah, this function returns 0 on success, 1 on failure. probably should fix that in another PR and change it to a bool